strangerstudios/paid-memberships-pro
# Widget Login Form Issues

### All Submissions:






### Changes proposed in this Pull Request:

Adjusted sanitization around the widget login form as multiple layers of escaping resulted in the fields being removed.

### How to test the changes in this Pull Request:

1. Add the Login Widget to a sidebar or widget field
2. The login form fields should still show and work as expected

### Other information:




### Changelog entry

Improvements to escaping contents of login form widget
